tests/_test_core_strings.py

```diff
@@ -67,6 +69,9 @@
     ]);
 }}
 </script>
+
+
```

Comment:
No, these are not intentional, those are a consequence of the templates
leaving empty lines after all the `{%....%}` statements.

On (Jun-28-16|10:41), Antonio Gonzalez wrote:
> > @@ -67,6 +69,9 @@
> >      ]);
> >  }}
> >  </script>
> > +
> > +
> 
> Do we really need these new lines?
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/510/files/378f456b29428865360dc7c3ad7a0beaed9e08b2#r68805659